libs/shared/styles/tailwind.config.js

```diff
@@ -0,0 +1,326 @@
+p/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
@StaberindeZA , correct me if I am wrong, but I think shared-styles is missing its `postcss.config.js` file and the following in this file:
```
const { createGlobPatternsForDependencies } = require('@nx/react/tailwind');
const { join } = require('path');
```
and then 
```
content: [
    join(
      __dirname,
      '{src,pages,components,app}/**/*!(*.stories|*.spec).{ts,tsx,html}'
    ),
    ...createGlobPatternsForDependencies(__dirname),
  ],
```
Reference: `apps/app1/tailwind.config.js` and `postcss.config.js` from [https://nx.dev/recipes/react/using-tailwind-css-in-react](https://nx.dev/recipes/react/using-tailwind-css-in-react)

Reply:
I'm not 100% I'm answering your question, but IMO these changes shouldn't be included in the shared `tailwind.config.js`

The additions you mention in steps 3 and 4, are mostly specific to Next, and wouldn't necessarily apply to other other apps that might use these shared styles.